lua/nvim-treesitter-textobjects/shared.lua

```diff
@@ -224,21 +224,24 @@ end
 ---@param range Range4
 ---@param row integer
 ---@param col integer
+---@param end_col_offset integer?
 ---@return boolean
-local function is_in_range(range, row, col)
+local function is_in_range(range, row, col, end_col_offset)
```

Comment:
Please add a brief comment on why the end_col_offset is needed (in what situation,) and a link to the issue.